### DISCUSSION TITLE
Delete copier.yaml

### DISCUSSION THREAD

**Discussion by gnawin** [TulipaEnergy/TulipaPlots.jl#17] [General]
Copier did an automatic update that failed (https://github.com/TulipaEnergy/TulipaPlots.jl/actions/runs/8000736445/job/21850654137#step:2:14).

The reason is that it misses a token as input.

There are two things I can do:

- Delete this file. I checked and we don't have this file in our main repo.
- Add my token, and hopefully this will work.

What do you suggest @abelsiqueira?

**Comment by abelsiqueira** [comment]
Can you try adding the token? There should be steps in COPIERTemplate.jl for that.

**Comment by gnawin** [comment]
@abelsiqueira 
Yeah, true I did not realize that. That was the step I skipped in https://github.com/abelsiqueira/COPIERTemplate.jl, because it did not work. We also discussed that the other day.

> (Step 5) Create a Personal Access Token to be used by the Copier workflow.
> 1. Go to https://github.com/settings/tokens.
> 2. Create a token with "Content", "Pull-request", and "Workflows" permissions.
> 3. Copy the Token.
> 4. Go to your YOUR_PACKAGE_URL/settings/secrets/actions.
> 5. Create a "New repository secret" named COPIER_PAT.

I could not do this

> Create a token with "Content", "Pull-request", and "Workflows" permissions.

Because I can only find "Workflows" permission. Is this enough?

**Comment by abelsiqueira** [comment]
You have to select the Tulipa repo first, and then the "repository permissions" appear. We missed that the other day, I think. This is for fine-grained tokens, btw.

**Comment by gnawin** [comment]
I followed the instructions, and rerun the action for Copier update, then I got an error "The process '/usr/bin/git' failed with exit code 128". After some googling, they suggest that there may still be the access issue.

Is it related to the token or something else?

FYI:
[my token]
![image](https://github.com/TulipaEnergy/TulipaPlots.jl/assets/125902905/0249e8d9-e7a7-4b9c-9e8f-9b1bb70644a4)

[my repo secret with the token copied in]
![image](https://github.com/TulipaEnergy/TulipaPlots.jl/assets/125902905/06be5c81-c411-49e0-b389-5647c170e6a0)

**Comment by abelsiqueira** [comment]
Good question, I don't really know. @suvayu, did you setup this for TulipaIO?

**Comment by suvayu** [comment]
I think I forgot the steps after pushing to GH :confounded:.  The Copier update jobs are failing similarly.
